presto-mysql/src/main/java/com/facebook/presto/plugin/mysql/MySqlClient.java

```diff
@@ -53,14 +54,15 @@
 import static com.facebook.presto.spi.StandardErrorCode.ALREADY_EXISTS;
 import static com.facebook.presto.spi.StandardErrorCode.NOT_SUPPORTED;
 import static com.google.common.util.concurrent.MoreExecutors.directExecutor;
-import static com.mysql.jdbc.SQLError.SQL_STATE_ER_TABLE_EXISTS_ERROR;
-import static com.mysql.jdbc.SQLError.SQL_STATE_SYNTAX_ERROR;
 import static java.lang.String.format;
 import static java.util.Locale.ENGLISH;
 
 public class MySqlClient
         extends BaseJdbcClient
 {
+    private static final String SQL_STATE_ER_TABLE_EXISTS_ERROR = "42S01";
+    //Mapping the MySql error numbers to JDBC SqlState Codes. See: https://dev.mysql.com/doc/connector-j/en/connector-j-reference-error-sqlstates.html.
```

Comment:
Can you please move this comment above the variable. And use javadoc comment formatting.
```suggestion
    /** 
    * Error code corresponding to code thrown when a table already exists.  The code is derived from the MySQL documentation. See {@link https://dev.mysql.com/doc/connector-j/en/connector-j-reference-error-sqlstates.html}
    */
```

Reply:
@ZacBlanco , thank you for the feedback. I made the changes . However , I could not use `See {@link https:...} ` as I was encountering an error `Cannot resolve symbol 'https'` . Upon a little digging , I found that Javadoc's `{@link}` is not meant for external URLs, so using `See {@link https:...} `will cause an error. I went with using `@see <a href ..>`instead for linking to external URLs. Please do let me know if this is okay and if there's something you'd want me to change/edit further.

Reply:
Thanks - sorry I forgot the link syntax for javadoc 😅 . This new version is much better